buildpacks/maven/CHANGELOG.md

```diff
@@ -4,9 +4,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [1.0.2] 2022/07/29
-
```

Comment:
These lines shouldn't be deleted - the release did occur.

Just checking you are reviewing the PR's "files changed" view after making changes, before requesting review? I often find I only spot typos/issues once looking at that view, and by checking it pre-emptively you save the review round trip time (as well as the time of the reviewer) :-)